src/utils/puns.ts

```diff
@@ -13,7 +13,12 @@ const CLEAN_RUN_PUNS = [
   'Fin-tastic job!',
   'You are dolphinitely doing great!',
   "It doesn't get any betta than this!",
-  "You're piranha roll now!"
+  "You're piranha roll now!",
+  "Valedictorian of the School of FSH",
+  "If you're FSHing for compliments, you got it!",
+  'Tank you for the great job',
+  "We don't have to print these puns, but we do it for the halibut",
```

Comment:
This one exceeds the 36 character limit as well.

Reply:
fixed